Carthage/Checkouts/stackviewcontroller/StackViewController/StackViewContainer.swift

```diff
@@ -13,13 +13,13 @@ import UIKit
 /// using view controller composition to display content.
 open class StackViewContainer: UIView, UIScrollViewDelegate {
     /// The scroll view that is the superview of the stack view.
-    open let scrollView: AutoScrollView
+    public let scrollView: AutoScrollView
```

Comment:
Is this technically a breaking API change? If so, should we increment the pod version to reflect?